### PR TITLE
0.0.2 >> Refactor and new classes

### PR DESCRIPTION
**Pull request by ravescovi**
Testing PR capabilities of VSCode.

### CODE REVIEW

File: gladier/base.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#     container_location = None
+#     container_flags = None
+
+class GladierBaseTool(object):
+    """Gladier Defaults defines a common method of tying together
+    flows, funcx-functions, and default inputs for starting a flow."""
+
+    flow_definition = None
+    flow_input = dict()
+    funcx_functions = []
```
Comment (NickolausDS) [inline gladier/base.py:18]: Should Base Tools include an optional `funcx_container` that is `None` by default (simply runs the function without a container if None) but can be configured by the BaseTool if desired?

**Comment by ravescovi** [comment]
I moved all base imports to base.py classes and the definitions to its own class.py files.
I also moved anything not pure Gladier based to the utils folder and will phase out the gladier.tools in later PR.

File: gladier/base.py (hunk 1 above)
Comment (ravescovi) [reply gladier/base.py:18]: That was my idea.
For now the way that funcx have been dealing with the containers is the following. 
`tomo_cont_id = fxc.register_container(location=os.path.join(conf['cont_dir'],container_name), container_type='singularity') 
download_cont_fxid = fxc.register_function(download_cont)
recon_fxid = fxc.register_function(tomo_recon, container_uuid=tomo_cont_id)`

so basically. if container_uuid is None, it does not register anything and treats as a normal funcx function. The problem is this is not on the master branch yet.

**Comment by ravescovi** [comment]
Should we also already kill the examples? on this PR?

**Review by ravescovi — COMMENTED** [review]
@NickolausDS 
Why do you need this here?
gladier/utils/dynamic_imports.py

**Pull request closed without merging**